tests/tests_suite_test.go

```diff
@@ -88,6 +88,19 @@ func (s *newSspStrategy) Init() {
 	Eventually(func() error {
 		return apiClient.Create(ctx, newSsp)
 	}, timeout, time.Second).ShouldNot(HaveOccurred())
+
+	Eventually(func() error {
+		return apiClient.Create(ctx, getTemplateWithCurrentVersion(strategy.GetTemplatesNamespace(), "v0.11.3"))
+	}, timeout, time.Second).ShouldNot(HaveOccurred())
+
+	Eventually(func() error {
+		return apiClient.Create(ctx, getTemplateWithOldVersion("1", strategy.GetTemplatesNamespace(), "v0.0.1"))
+	}, timeout, time.Second).ShouldNot(HaveOccurred())
+
+	Eventually(func() error {
+		return apiClient.Create(ctx, getTemplateWithOldVersion("2", strategy.GetTemplatesNamespace(), "v0.0.1"))
+	}, timeout, time.Second).ShouldNot(HaveOccurred())
+
```

Comment:
It may be more clear if these lines that create templates were moved somewhere closer to the test case. Maybe to `BeforeEach()` or directly inside the test. But then the test would need to trigger reconciliation, for example by modifying a watched resource. I leave it up to you to decide.

In any case, these templates needs to be removed either after the test or on cleanup.

Reply:
I would prefer to create in `BeforeSuite`, because they should represent templates, which were in cluster before ssp operator was deployed.
They will be deleted during namespace deletion https://github.com/kubevirt/ssp-operator/blob/master/tests/tests_suite_test.go#L109

Reply:
Ok, then please add to code to create the templates also to the `existingSspStrategy`.